metrics-dropwizard/src/main/scala/com/mesosphere/usi/metrics/dropwizard/reporters/DataDogAPIReporter.scala

```diff
@@ -86,10 +105,22 @@ class DataDogAPIReporter(settings: DataDogApiReporterSettings, registry: MetricR
     reportMetric(buffer, name, counter.getCount.toString, timestamp, "gauge")
 
   private val histogramSnapshotSuffixes =
-    Seq("min", "average", "median", "75percentile", "95percentile", "98percentile",
-      "99percentile", "999percentile", "max", "stddev")
-  private def reportSnapshot(buffer: StringBuilder, name: String, snapshot: Snapshot, timestamp: Long,
-    scaleMetrics: Boolean): Unit = {
+    Seq("min",
+        "average",
```

Comment:
I tend to agree with @kensipe on indenting further args this way; with really-long-method-names this can cause a lot of our code to be very far to the right.